service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/application/ZoneApplication.java

```diff
@@ -1,5 +1,5 @@
 // Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
-package com.yahoo.vespa.service.monitor.internal;
+package com.yahoo.vespa.service.monitor.internal.application;
```

Comment:
These applications should no longer be placed in an internal package

Reply:
Moved to own package